docs/source/build-run-openvino.md

```diff
@@ -0,0 +1,169 @@
+:::
+::::
+
+## Introduction to OpenVINO
+
+[OpenVINO](https://www.intel.com/content/www/us/en/developer/tools/openvino-toolkit/overview.html) is an open-source toolkit designed to enhance AI inference on Intel hardware by reducing latency and increasing throughput while preserving accuracy. It optimizes hardware utilization and simplifies AI development and deep learning integration across domains such as computer vision, large language models (LLMs), and generative AI.
+
+OpenVINO is integrated as an Executorch delegate to accelerate AI applications deployed with Executorch APIs. 
+
+## Supported Hardware
+
+OpenVINO backend supports the following hardware:
+
+- Intel CPUs
+- Intel integrated GPUs
+- Intel discrete GPUs
+- Intel NPUs
+
+## Instructions for Building OpenVINO Backend
+
+### Prerequisites
+
+Before you begin, ensure you have openvino installed and configured on your system:
+
+#### TODO: Add instructions for support with OpenVINO release package
+
+```bash
+git clone https://github.com/openvinotoolkit/openvino.git
+cd openvino && git checkout releases/2025/1
+git submodule update --init --recursive
+mkdir build && cd build
+cmake .. -DCMAKE_BUILD_TYPE=Release -DENABLE_PYTHON=ON
+make -j<N>
+
+cd ..
+cmake --install build --prefix <your_preferred_install_location>
+cd <your_preferred_install_location>
+source setupvars.sh
+```
+
+### Setup
+
+Follow the steps below to setup your build environment:
+
+1. **Setup ExecuTorch Environment**: Refer to the [Environment Setup](https://pytorch.org/executorch/stable/getting-started-setup#environment-setup) guide for detailed instructions on setting up the ExecuTorch environment.
+
+2. **Setup OpenVINO Backend Environment**
+- Install the dependent libs. Ensure that you are inside `executorch/backends/openvino/` directory
+   ```bash
+   pip install -r requirements.txt
+   ```
+
+3. Navigate to `scripts/` directory.
+
+4. **Build OpenVINO Backend**: Once the prerequisites are in place, run the `openvino_build.sh` script to start the build process, OpenVINO backend will be built under `cmake-openvino-out/backends/openvino/` as `libopenvino_backend.so`
+
+   ```bash
+   ./openvino_build.sh
+   ```
+
+## Build Instructions for Examples
+
+### AOT step:
+Refer to the [README.md](../../examples/openvino/aot/README.md) in the `executorch/examples/openvino/aot` folder for detailed instructions on exporting deep learning models from various model suites (TIMM, Torchvision, Hugging Face) to openvino backend using Executorch. Users can dynamically specify the model, input shape, and target device. 
+
+Below is an example to export a ResNet50 model from Torchvision model suite for CPU device with an input shape of `[1, 3, 256, 256]`
+
+```bash
+cd executorch/examples/openvino/aot
+python aot_openvino_compiler.py --suite torchvision --model resnet50 --input_shape "(1, 3, 256, 256)" --device CPU
+```
+The exported model will be saved as 'resnet50.pte' in the current directory.
+
+#### **Arguments**
```

Comment:
I dont think you need to list detailed oarguments for the scripts in examples folder. I would move this to read me in the examples/openvino

Reply:
Removed arguments section from this file.